composer.json

```diff
@@ -15,13 +15,17 @@
         }
     },
     "require": {
-        "illuminate/container": "~5.0",
-        "illuminate/support": "~5.0",
-        "illuminate/view": "~5.0",
-        "symfony/console": "~2.5",
-        "mockery/mockery": "^0.9.4"
+        "illuminate/container": "^5.2",
+        "illuminate/support": "^5.2",
+        "illuminate/view": "^5.2",
+        "symfony/console": "^2.5",
+        "mockery/mockery": "^0.9.4",
+        "mnapoli/front-yaml": "^1.5"
```

Comment:
FrontYAML is a little wrapper around Symfony YAML and Parsedown specifically for parsing Markdown files with Frontmatter, not just a YAML parser, so if I switched to something else I'd have to split out the front matter myself to parse separately, blah blah.